examples/research_projects/codeparrot/README.md

```diff
@@ -12,7 +12,7 @@ This is an open-source effort to train and evaluate code generation models. Code
 - continuously push checkpoints to the hub with `huggingface_hub`
 - stream the dataset with `datasets` during training to avoid disk bottlenecks
 - apply the `code_eval` metric in `datasets` to evaluate on [OpenAI's _HumanEval_ benchmark](https://huggingface.co/datasets/openai_humaneval)
-
+- showcase examples for downstream tasks with code models in [examples](https://github.com/huggingface/transformers/tree/main/examples/research_projects/codeparrot/examples) folder
```

Comment:
maybe we can say what examples we show there. should we also add the code for the text2py and py2text here?

Reply:
Ok but the text2py and py2text examples use a similar script to the one for pretraining codeparrot just with a different dataset and model checkpoint, maybe I can just mention that in the README?